hive/lib/src/backend/stub/storage_backend_memory.dart

```diff
@@ -0,0 +1,127 @@
+import 'dart:typed_data';
+
+import 'package:hive/hive.dart';
+import 'package:hive/src/backend/storage_backend.dart';
+import 'package:hive/src/backend/vm/read_write_sync.dart';
+import 'package:hive/src/binary/binary_reader_impl.dart';
+import 'package:hive/src/binary/binary_writer_impl.dart';
+import 'package:hive/src/binary/frame.dart';
+import 'package:hive/src/binary/frame_helper.dart';
+import 'package:hive/src/box/keystore.dart';
+
+/// In-memory Storage backend
```

Comment:
May I ask for one or two lines more? I know, Hive's sources are only coarsely documented up to now, but unless we try add all relevant information to any public member in new contributions, this won't change.

Just some lines of a) what it does b) how to use it and c) how it's done. Thanks!